app/pb_kits/playbook/pb_currency/currency.rb

```diff
@@ -61,20 +55,32 @@ def value_size
         size == "lg" ? 1 : 2
       end
 
+      def formatted_value
+        number_to_currency(configured_value)
+      end
+
       def value
         if is_set? configured_value
-          pb_title = Playbook::PbTitle::Title.new(size: value_size, text: configured_value, classname: "pb_currency_value")
+          display_value = formatted_value[1...-2]
+          pb_title = Playbook::PbTitle::Title.new(size: value_size, text: display_value, classname: "pb_currency_value")
           ApplicationController.renderer.render(partial: pb_title, as: :object)
```

Comment:
As we've been moving over kits to the new format, we've been removing all `ApplicationController.renderer.render` invocations in preference to simply using `pb_rails` in the view partial. You're fine to keep this here, but wanted to give you a heads up. -- I guess as your team enhances more kits, we should try to move them into the view file for consistency.